source/Grid/Grid.js

```diff
@@ -204,6 +204,8 @@ export default class Grid extends Component {
 
     // See defaultCellRangeRenderer() for more information on the usage of this cache
     this._cellCache = {}
+
+    this._scrollbarSize = getScrollbarSize()
```

Comment:
I don't believe this is a good idea. I think it could cause problems with server-side rendering.